packages/react/src/components/Checkbox/Checkbox.js

```diff
@@ -127,7 +131,10 @@ Checkbox.propTypes = {
   /**
    * The CSS class name to be placed on the wrapping element
    */
-  wrapperClassName: PropTypes.string,
+  wrapperClassName: deprecate(
```

Comment:
If people followed the deprecation notice and used `className` would the logic above pick up that `wrapperClassName` is not being used and it uses `className` instead? Wasn't 100% sure so figured I'd ask 👀

Reply:
Good catch! I can update the wording to be a bit more clear. How's this sound?

"The prop `wrapperClassName` for Checkbox will be deprecated in V11 in favor of `className`. `className` will then be placed on the outer wrapper."

Reply:
Ah so sorry, totally misread what was happening. For cases where we have both a `className` prop and a class name to target an inner node (like `wrapperClassName` in this case) do you think we should give people a way to target the inner node? Or is it better to constrain them to the `className` on the outermost node? I think that's what tripped me up, I thought we were adding in `className` as a replacement to `wrapperClassName` but I didn't realize that the component already had a `className` prop

Reply:
I figured it would be best that if a user passes in `className`, it is always placed on the outer node across the repo. That would still give them access to the inner nodes via the class they pass in. It would also reduce these `wrapperClassName` props that were added later on because we were placing `className` on the inner nodes and people had no way to target the outer nodes.

Reply:
That way the experience is consistent across the repo. You pass in `className`, it always goes on the outer node and you have access to all internal nodes

Reply:
Sounds good! So in this case, would the CSS they would write be: `.my-custom-class .bx--form-item` to target what `wrapperClasses` sets or are you thinking it'd be something else?

Reply:
I think it'd be `.my-custom-class.bx--form-item` since they're both placed on the same elements but yeah that's what I was thinking 👍🏻